src/middlewared/middlewared/utils/__init__.py

```diff
@@ -331,6 +331,10 @@ def sw_version_is_stable():
     return version['stable']
 
 
+def get_cpu_model():
```

Comment:
I am still not convinced this is necessary

Reply:
I wasn't sure about it myself. Should we use `osc` directly in system plugin ?

Reply:
Yes, we import from osc directly